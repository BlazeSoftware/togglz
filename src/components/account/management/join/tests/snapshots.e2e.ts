import snapIt from '../../../../../test/snap-it';

describe('join', () => {
  snapIt('account-join')('renders correctly', '<account-join></account-join>');
});
