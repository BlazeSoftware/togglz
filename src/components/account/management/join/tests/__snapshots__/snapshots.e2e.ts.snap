// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`join renders correctly 1`] = `"<account-join class=\\"hydrated\\"><div class=\\"o-container o-container--xsmall u-window-box-medium\\"><blaze-alert class=\\"hydrated\\"><!----><div role=\\"alert\\" hidden=\\"\\" class=\\"c-alert u-display-none\\"><div></div><div></div></div></blaze-alert><blaze-card class=\\"hydrated\\"><!----><div class=\\"c-card\\"><form><blaze-card-header class=\\"hydrated\\"><!----><header class=\\"c-card__header\\"><h2 class=\\"c-heading\\">Create a new account</h2></header></blaze-card-header><blaze-card-body class=\\"hydrated\\"><!----><div class=\\"c-card__body\\"><label class=\\"c-label o-form-element\\">Email address:<div class=\\"o-field o-field--icon-left\\"><i aria-hidden=\\"\\" class=\\"fa-fw fas fa-at c-icon\\"></i><input type=\\"email\\" class=\\"c-field c-field--label\\" required=\\"\\"></div></label><label class=\\"o-form-element c-label\\">Password:<div class=\\"c-input-group c-input-group--label\\"><div class=\\"o-field o-field--icon-left\\"><i aria-hidden=\\"\\" class=\\"fa-fw fas fa-lock c-icon\\"></i><input type=\\"password\\" class=\\"c-field\\" required=\\"\\" minlength=\\"6\\"></div><button type=\\"button\\" class=\\"c-button c-button--ghost c-button--brand\\">Show</button></div></label></div></blaze-card-body><blaze-card-footer class=\\"hydrated\\"><!----><footer class=\\"c-card__footer\\"><button class=\\"c-button c-button--block c-button--brand\\"><span class=\\"c-button__icon-left\\" aria-hidden=\\"\\"><i aria-hidden=\\"\\" class=\\"fa-fw fas fa-user-plus\\"></i></span>Create new account</button></footer></blaze-card-footer></form></div></blaze-card></div></account-join>"`;
